internal/gqlclient/gqlclient.go

```diff
@@ -307,21 +258,21 @@ func (c *Client) createMultiPartUploadRequest(gqlReq RequestWithFiles) (*http.Re
 		for n, file := range gqlReq.Files() {
 			part, err := mw.CreateFormFile(fmt.Sprintf("%d", n), file.Name)
 			if err != nil {
-				multilog.Error("Could not create form file: %v", err)
+				reqErrChan <- errs.Wrap(err, "Could not create form file")
 				return
 			}
 
 			_, err = io.Copy(part, file.R)
 			if err != nil {
-				multilog.Error("Could not read file: %v", err)
+				reqErrChan <- errs.Wrap(err, "Could not read file")
 				return
 			}
 		}
 	}()
 
 	varJson, err := json.Marshal(vars)
 	if err != nil {
-		return nil, errs.Wrap(err, "Could not marshal vars")
+		return errs.Wrap(err, "Could not marshal vars")
```

Comment:
Please move this above the goroutine, cause otherwise if and when this triggers the goroutine would be left dangling.